### PR TITLE
SIG Security 2021 annual report

### LINKED ISSUE #6380: 2021 Annual Report: SIG Security
Chairs: @IanColdwater @tabbysable 
Liaison: @parispittman

Actions for the chair/organizer of the community group:
- [ ] Consult your community group to complete draft of report
- [ ] If needed, consult with Steering Committee liaison on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your Steering Committee liaison for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, Steering Committee will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

Key dates:
- Initial PR to communtiy repo should be opened by February 14, 2022
- PR should be reviewed and merged by March 1st, 2022

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

Your group's report template is available here: https://git.k8s.io/community/sig-security/annual-report-2021.md

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liaison directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/sig security
/committee steering

### PR DESCRIPTION
**Pull request by IanColdwater**
**Which issue(s) this PR fixes**:

Fixes #6380

First draft of SIG Security 2021 annual report. WILLFIX

### CODE REVIEW

File: sig-security/annual-report-2021.md [hunk 1]
```diff
@@ -1,143 +1,134 @@
...
 
-2. What initiatives are you working on that aren't being tracked in KEPs?
+  Our Docs subproject has made Kubernetes security more approachable through blogs and tutorials such as [A Closer Look at NSA/CISA Kubernetes Hardening Guidance](https://kubernetes.io/blog/2021/10/05/nsa-cisa-kubernetes-hardening-guidance/), [Securing Admission Controllers | Kubernetes](https://kubernetes.io/blog/2022/01/19/secure-your-admission-controllers-and-webhooks/), [Apply Pod Security Standards at the Cluster Level](https://kubernetes.io/docs/tutorials/security/cluster-level-pss/), and [Apply Pod Security Standards at the Namespace Level](https://kubernetes.io/docs/tutorials/security/ns-level-pss/).
```
Comment (tabbysable) [inline sig-security/annual-report-2021.md:9]:
```suggestion
  Our Docs subproject has made Kubernetes security more approachable through blogs and tutorials such as [A Closer Look at NSA/CISA Kubernetes Hardening Guidance](https://kubernetes.io/blog/2021/10/05/nsa-cisa-kubernetes-hardening-guidance/), [Securing Admission Controllers](https://kubernetes.io/blog/2022/01/19/secure-your-admission-controllers-and-webhooks/), [Apply Pod Security Standards at the Cluster Level](https://kubernetes.io/docs/tutorials/security/cluster-level-pss/), and [Apply Pod Security Standards at the Namespace Level](https://kubernetes.io/docs/tutorials/security/ns-level-pss/).
```

File: sig-security/annual-report-2021.md [hunk 2]
```diff
@@ -1,143 +1,134 @@
...
+
+  Our Tooling subproject is working to prototype, define policy for, and implement [vulnerability scanning for build time dependencies and container images](https://github.com/kubernetes/sig-security/issues/3]. This initiative involves collaborating with several other SIGs, applying SIG Security’s specialized experience with these tools to produce meaningful improvements without burying other SIGs in additional work.
```
Comment (tabbysable) [inline sig-security/annual-report-2021.md:21]:
```suggestion
  Our Tooling subproject is working to prototype, define policy for, and implement [vulnerability scanning for build time dependencies and container images](https://github.com/kubernetes/sig-security/issues/3). This initiative involves collaborating with several other SIGs, applying SIG Security’s specialized experience with these tools to produce meaningful improvements without burying other SIGs in additional work.
```

File: sig-security/annual-report-2021.md [hunk 3]
```diff
@@ -1,143 +1,134 @@
...
-   does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
+**3. Does your [CONTRIBUTING.md] help _new_ contributors engage with your group specifically by pointing to activities or programs that provide useful context or allow easy participation?**
 
-   -
 
-5. Does the group have contributors from multiple companies/affiliations?
+- We have known for some time that SIG Security needs a CONTRIBUTING.MD document that more accurately reflects how we do things. SIG Security takes a community-building approach to improving Kubernetes security: we intentionally make a space where someone can begin contributing by sharing their thoughts and ideas in a meeting or Slack message. The CONTRIBUTING.MD document we have in progress reflects this. It encourages participation by showing up and links to the general Kubernetes contributor guides, which are still applicable.
 
-   -
+**4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide], does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?**
```
Comment (tabbysable) [inline sig-security/annual-report-2021.md:61]:
```suggestion
**4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide], does your [CONTRIBUTING.md] document those to help _existing_ contributors grow throughout the [contributor ladder]?**
```

File: sig-security/annual-report-2021.md [hunk 4]
```diff
@@ -1,143 +1,134 @@
...
+    (or created if missing and your contributor steps and experience are different or more
+    in-depth than the documentation listed in the general [contributor guide] and [devel] folder.)
+- [x] Subprojects list and linked OWNERS files in [sigs.yaml] reviewed for accuracy and updated if needed
+- [x] SIG leaders (chairs, tech leads, and subproject owners) in [sigs.yaml] are accurate and active, and updated if needed
+- [x] Meeting notes and recordings for 2021 are linked from [README.md] and updated/uploaded if needed
+- [x] Did you have community-wide updates in 2021 (e.g. community meetings, kubecon, or kubernetes-dev@ emails)? Links to email, slides, or recordings:
+    - KubeCon EU: [Get In Containerds, We’re Going Securing: Kubernetes SIG Security is Here!](https://www.youtube.com/watch?v=0_s6zkyRpME)
+    - KubeCon NA: [Security Through Transparency: Kubernetes SIG Security Update](https://www.youtube.com/watch?v=O5Wy7zSigOU)
+
+[CONTRIBUTING.md]: https://git.k8s.io/community/$sig-id/CONTRIBUTING.md
```
Comment (tabbysable) [inline sig-security/annual-report-2021.md:128]:
```suggestion
[CONTRIBUTING.md]: https://git.k8s.io/community/sig-security/CONTRIBUTING.md
```

File: sig-security/annual-report-2021.md [hunk 5]
```diff
@@ -1,143 +1,134 @@
...
 [contributor ladder]: https://git.k8s.io/community/community-membership.md
 [sig-governance.md]: https://git.k8s.io/community/committee-steering/governance/sig-governance.md
-[README.md]: https://git.k8s.io/community/sig-security/README.md
+[README.md]: https://git.k8s.io/community/$sig-id/README.md
```
Comment (tabbysable) [inline sig-security/annual-report-2021.md:131]:
```suggestion
[README.md]: https://git.k8s.io/community/sig-security/README.md
```

**Comment by tabbysable** [comment]
I read through the rendered preview and picked a bunch of Markdown nits using GitHub suggestion mode.

File: sig-security/annual-report-2021.md [hunk 6]
```diff
@@ -1,143 +1,134 @@
...
 
-3. KEP work in 2021 (1.x, 1.y, 1.z):
+  SIG Security does things differently than a lot of other SIGs. We don’t own code. We work hard to cultivate and grow an inclusive, welcoming environment where everyone, from the newest contributor to the most experienced hacker, can come share their ideas and contribute to the security of the project. This is designed to ripple outward whether or not we are involved, encouraging and facilitating internal collaboration across SIGs and engagement with the external security community. We’re really proud of how far we’ve come, and we’re excited to continue leading by example into 2022 and beyond!
 
-<!--
-In future, this will be generated from kubernetes/enhancements kep.yaml files
-1. with SIG as owning-sig or in participating-sigs
-2. listing 1.x, 1.y, or 1.z in milestones or in latest-milestone
--->
+**2. What initiatives are you working on that aren't being tracked in KEPs?**
```
Comment (parispittman) [inline sig-security/annual-report-2021.md:15]: add contact info as links or however you see fit for the subproject blurbs. example [tooling subproject](link to slack channel)

File: sig-security/annual-report-2021.md [hunk 7]
```diff
@@ -1,143 +1,134 @@
...
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+      - [2763 - Ambient Capabilities](https://git.k8s.io/enhancements/keps/sig-security2763-ambient-capabilities/README.md)
 
 ## Project health
 
-1. What areas and/or subprojects does your group need the most help with?
-   Any areas with 2 or fewer OWNERs? (link to more details)
+**1. What areas and/or subprojects does your group need the most help with?**
+
+- Our Docs subproject is always looking for security-minded contributors of all experience levels to share their learning and knowledge with the community! This subproject has consistently been a place where people [merge](https://github.com/kubernetes/website/pull/28248) their [first](https://github.com/kubernetes/website/pull/31518) Kubernetes PRs.There’s always room for continuous improvement in our documentation, and contributing to this provides an opportunity to learn more about Kubernetes security while helping everyone run their clusters more safely. We’re really proud of the way Docs encourages and welcomes new contributors, and we’d love to encourage you to become a part of it!
```
Comment (parispittman) [inline sig-security/annual-report-2021.md:43]: at the end of this, how can someone be apart of it - show up at a meeting? where? slack channel? same q

**Review by parispittman — CHANGES_REQUESTED** [review]
this looks GREAT!! thank you so much for your thoughtful responses. You all have really grown since the last time we did this. :)  I have small nits and in the meantime, feel free to email this PR to your SIG mailing list/slack with whatever your usual lazy consensus period (72 hours?) is for last calls for comments before merging.

File: sig-security/annual-report-2021.md [hunk 8]
```diff
@@ -1,143 +1,134 @@
...
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
+      - [2579 - PSP Replacement Policy](https://git.k8s.io/enhancements/keps/sig-auth/2579-psp-replacement/README.m) - 1.23
+      - [1933 - Defend against logging secrets via static analysis](https://git.k8s.io/enhancements/keps/sig-security/1933-secret-logging-static-analysis/README.md) - 1.21
    - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
+       - [2579 - PSP Replacement Policy](https://git.k8s.io/enhancements/keps/sig-auth/2579-psp-replacement/README.md) - 1.22
    - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+      - [2763 - Ambient Capabilities](https://git.k8s.io/enhancements/keps/sig-security2763-ambient-capabilities/README.md)
```
Comment (christophetd) [inline sig-security/annual-report-2021.md:37]:
```suggestion
      - [2763 - Ambient Capabilities](https://github.com/kubernetes/enhancements/blob/master/keps/sig-security/2763-ambient-capabilities/README.md)
```

Comment (christophetd) [inline sig-security/annual-report-2021.md:37]:
```suggestion
      - [2763 - Ambient Capabilities](https://git.k8s.io/enhancements/keps/sig-security/2763-ambient-capabilities/README.md)
```

File: sig-security/annual-report-2021.md [hunk 9]
```diff
@@ -1,143 +1,134 @@
...
+  Our Docs subproject has made Kubernetes security more approachable through blogs and tutorials such as [A Closer Look at NSA/CISA Kubernetes Hardening Guidance](https://kubernetes.io/blog/2021/10/05/nsa-cisa-kubernetes-hardening-guidance/), [Securing Admission Controllers | Kubernetes](https://kubernetes.io/blog/2022/01/19/secure-your-admission-controllers-and-webhooks/), [Apply Pod Security Standards at the Cluster Level](https://kubernetes.io/docs/tutorials/security/cluster-level-pss/), and [Apply Pod Security Standards at the Namespace Level](https://kubernetes.io/docs/tutorials/security/ns-level-pss/).
 
-   -
-   -
-   -
+  Our Tooling subproject got off the ground in 2021 and is now flying high, [standardizing Kubernetes CVE reporting](https://github.com/kubernetes/sig-security/issues/1), [prototyping vulnerability scanning of Kubernetes artifacts](https://github.com/kubernetes/sig-security/issues/3), and [hosting](https://www.youtube.com/watch?v=zB1-7NLsfps) [learning](https://www.youtube.com/watch?v=o-E6aoKmznY) [sessions](https://www.youtube.com/watch?v=MWAb63gf3gs) which have been viewed hundreds of times.
```
Comment (tabbysable) [inline sig-security/annual-report-2021.md:11]:
```suggestion
  Our [Tooling subproject](https://kubernetes.slack.com/archives/C01CUSVMHPY) got off the ground in 2021 and is now flying high, [standardizing Kubernetes CVE reporting](https://github.com/kubernetes/sig-security/issues/1), [prototyping vulnerability scanning of Kubernetes artifacts](https://github.com/kubernetes/sig-security/issues/3), and [hosting](https://www.youtube.com/watch?v=zB1-7NLsfps) [learning](https://www.youtube.com/watch?v=o-E6aoKmznY) [sessions](https://www.youtube.com/watch?v=MWAb63gf3gs) which have been viewed hundreds of times.
```

File: sig-security/annual-report-2021.md [hunk 10]
```diff
@@ -1,143 +1,134 @@
...
+
+  Our Tooling subproject is working to prototype, define policy for, and implement [vulnerability scanning for build time dependencies and container images](https://github.com/kubernetes/sig-security/issues/3]. This initiative involves collaborating with several other SIGs, applying SIG Security’s specialized experience with these tools to produce meaningful improvements without burying other SIGs in additional work.
+
+  Our Third-Party Audit subproject continues to administer their namesake initiative. In 2021 they published a [Request for Proposals](https://github.com/kubernetes/sig-security/blob/main/sig-security-external-audit/security-audit-2021/RFP.md), evaluated the vendor responses, and began to negotiate contract terms with the selected vendor. As of this writing, they have [announced the vendor selection](https://github.com/kubernetes/sig-security/blob/main/sig-security-external-audit/security-audit-2021/RFP_Decision.md) and are making arrangements for work to begin.
```
Comment (tabbysable) [inline sig-security/annual-report-2021.md:23]:
```suggestion
  Our [Third-Party Audit subproject](https://github.com/kubernetes/sig-security/tree/main/sig-security-external-audit) continues to administer their namesake initiative. In 2021 they published a [Request for Proposals](https://github.com/kubernetes/sig-security/blob/main/sig-security-external-audit/security-audit-2021/RFP.md), evaluated the vendor responses, and began to negotiate contract terms with the selected vendor. As of this writing, they have [announced the vendor selection](https://github.com/kubernetes/sig-security/blob/main/sig-security-external-audit/security-audit-2021/RFP_Decision.md) and are making arrangements for work to begin.
```

File: sig-security/annual-report-2021.md [hunk 11]
```diff
@@ -1,143 +1,134 @@
...
+
+  Our Security Docs subproject works in collaboration with SIG Docs and others on its continuing initiatives: to improve security representation and content in Kubernetes documentation, to communicate information to help end users and others keep themselves safer, and to help foster and grow new contributors. They have been prolifically publishing useful information such as the blog posts shouted out in the beginning of this report, and highlighting good first issues for new contributors to start diving in.
```
Comment (tabbysable) [inline sig-security/annual-report-2021.md:25]:
```suggestion
  Our [Security Docs subproject](https://kubernetes.slack.com/archives/C01D8R7ACQ2) works in collaboration with SIG Docs and others on its continuing initiatives: to improve security representation and content in Kubernetes documentation, to communicate information to help end users and others keep themselves safer, and to help foster and grow new contributors. They have been prolifically publishing useful information such as the blog posts shouted out in the beginning of this report, and highlighting good first issues for new contributors to start diving in.
```

File: sig-security/annual-report-2021.md (hunk 6 above)
Comment (tabbysable) [reply sig-security/annual-report-2021.md:15]: Done.

File: sig-security/annual-report-2021.md (hunk 7 above)
Comment (tabbysable) [inline sig-security/annual-report-2021.md:43]:
```suggestion
- Our [Docs subproject](https://kubernetes.slack.com/archives/C01D8R7ACQ2) is always looking for security-minded contributors of all experience levels to share their learning and knowledge with the community! This subproject has consistently been a place where people [merge](https://github.com/kubernetes/website/pull/28248) their [first](https://github.com/kubernetes/website/pull/31518) Kubernetes PRs.There’s always room for continuous improvement in our documentation, and contributing to this provides an opportunity to learn more about Kubernetes security while helping everyone run their clusters more safely. We’re really proud of the way Docs encourages and welcomes new contributors, and we’d love to encourage you to become a part of it!
```

File: sig-security/annual-report-2021.md [hunk 12]
```diff
@@ -1,143 +1,134 @@
-# 2021 Annual Report: SIG Security
+# SIG Security - 2021 annual report
 
 ## Current initiatives
 
-1. What work did the SIG do this year that should be highlighted?
+**1. What work did the SIG do this year that should be highlighted?**
```
[the remark this answers is not in the document]
Comment (IanColdwater) [reply sig-security/annual-report-2021.md:5]: <3

**Comment by parispittman** [comment]
/lgtm

thanks team!

**Pull request merged**